### PR TITLE
polls fetching

### PR DESCRIPTION
**Pull request by dotslash227**
- GET Api's for fetching polls for user
- GET api for fetching polls of user's friends

### CODE REVIEW

File: server/routes/polls/polls.route.js [hunk 1]
```diff
@@ -99,4 +99,45 @@ router.put(
...
+router.get(
+  '/friends',
+  passport.authenticate('jwt', { session: true }),
+  (req, res) => {
+    const user = req.user;
+    const friends = user.friendIds;
+    const polls = [];
+
+    friends.map((friend) => {
+      Poll.find({ userId: friend._id }, (err, docs) => {
```
Comment (tgnemecek) [inline server/routes/polls/polls.route.js]: According to the user model, the friendIds is an array of objectIds:
`
  friendIds: [
    {
      type: mongoose.Schema.Types.ObjectId,
      required: true,
      ref: 'users',
    },
  ],
`
```suggestion
      Poll.find({ userId: friend }, (err, docs) => {
```

File: server/package.json [hunk 2]
```diff
@@ -24,6 +24,8 @@
     "multer-s3": "^2.9.0",
     "passport": "^0.4.1",
     "passport-jwt": "^4.0.0",
+    "react-dropzone": "^11.0.1",
```
Comment (tgnemecek) [inline server/package.json]: Shouldn't this be on the client instead?

**Review by tgnemecek — CHANGES_REQUESTED** [review]
Just the two comments I made, the rest is great!

File: server/package.json (hunk 2 above)
Comment (dotslash227) [reply server/package.json]: I might have installed it on the server end too, I'll fix it.

File: server/routes/polls/polls.route.js (hunk 1 above)
Comment (dotslash227) [reply server/routes/polls/polls.route.js]: Hmm. friend._id did work out for me. I'll test out your suggestion and commit. ETA: ~5 mins.

**Review by tgnemecek — APPROVED** [review]
Nice!

**Pull request merged**